response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/base.html" %}
+
+{% set subject = messages[-1]["subject"] or "No Subject" %}
```

Comment:
Not sure if you need the "No Subject" as I don't think we can send a message without a subject.
🎨 And I think you can set default like `{% set subject = messages[-1].get("subject", "No Subject") %}`